Gemfile

```diff
@@ -32,13 +32,14 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem 'factory_bot_rails'
+  gem 'ffaker'
```

Comment:
i dont recall putting ffaker for the article app but the shoulda matchers still worked tho

Reply:
i added this, ffaker is useful for generating realistic data.
https://github.com/ffaker/ffaker
So we can ask for random names, or other things then having to make them ourselves or create strange methods to generate realistic fake data